src/frequenz/lib/datetime_time_checker.py

```diff
@@ -0,0 +1,192 @@
+"""Pylint checker ensure correct usage of datetime module, especially around aware/naive objects."""
+
+from typing import Any
+
+import astroid  # type: ignore
+import pylint  # type: ignore
```

Comment:
Are you sure pylint itself needs this exclusion as well?

Reply:
Also importing should not have typing issues, I don't think any of those `type: ignore` should be there. If `astroid` is missing type info, then an ignore should be added to `pyproject.toml`.